Splitz/BillScanner/BillScannerViewController.swift

```diff
@@ -10,21 +10,28 @@ import UIKit
 struct BillScannnerConstants {
     var title = NSLocalizedString("BillScannerViewController_Title", comment: "")
     var manualEntryButtonTitle = NSLocalizedString("Enter amount manually", comment: "")
+    var scanningStatus = NSLocalizedString("Scanning the bill..", comment: "")
```

Comment:
Yooo mit man friend. scanningStatus = NSLocalizedString("Scanning the bill..", comment: ""), replace "scanning the bill